.ci/pipelines/e2e-tests-gke-k8s-versions.Jenkinsfile

```diff
@@ -46,7 +46,7 @@ pipeline {
                         }
                     }
                 }
-                stage("1.16") {
+                stage("1.18") {
```

Comment:
Same here, I think we should update `runWith(lib, failedTests, '1.16', "eck-gke16-${BUILD_NUMBER}-e2e")`